### PR TITLE
format: Search and collections bar styling

### PR DESCRIPTION
**Pull request by nikpap**
* New top bar styling.

Signed-off-by: Nick Papoutsis <admin@nils.gr>

### CODE REVIEW

File: inspirehep/base/templates/header.html [hunk 1]
```diff
@@ -43,245 +42,172 @@
...
+      </a>
+      {% if request.endpoint in ['search.search', 'record.metadata'] %}
+        <div id="top-search-bar-small" class="col-sm-5 {{collection.name}}">
+          {% include "search/form/form.html" %}
+        </div>
+      {% endif %}
+    </div>
+    {% if request.endpoint in ['search.search', 'record.metadata'] %}
+      <div id="top-search-bar-full" class="col-md-4 col-lg-5 {{collection.name}}">
+        {% include "search/form/form.html" %}
```
Comment (jmartinm) [inline inspirehep/base/templates/header.html]: Why is the search bar included twice?

File: inspirehep/base/static/less/base/header.less [hunk 2]
```diff
@@ -629,3 +596,173 @@ a.external-link::after {
...
+        }
+
+        &.data {
+            .collection-background-color(@data-collection);
+        }
+    }
+}
+
+// TO-DO: fix these..
+@media (min-width: 767px) {
```
Comment (jmartinm) [inline inspirehep/base/static/less/base/header.less]: would be good to use the standard break points that we use overall (with their variable name)

File: inspirehep/base/templates/search/searchbar_frame_base.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+{% extends "page.html" %}
```
Comment (jmartinm) [inline inspirehep/base/templates/search/searchbar_frame_base.html]: When is this template used?

File: inspirehep/base/templates/search/searchbar_frame_base.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{#
```
Comment (jmartinm) [inline inspirehep/base/templates/search/searchbar_frame_base.html]: When is this template used?

File: inspirehep/base/templates/header.html (hunk 1 above)
Comment (nikpap) [inline inspirehep/base/templates/header.html]: The `top-search-bar-small` is included in the `navbar-header` div, whereas the `top-search-bar-full` is outside it, I found this necessary to accommodate for the mobile view. Open to suggestions for improvement though!

File: inspirehep/base/templates/search/searchbar_frame_base.html (hunk 4 above)
Comment (nikpap) [inline inspirehep/base/templates/search/searchbar_frame_base.html]: I'm overwriting https://github.com/inspirehep/invenio-search/blob/master/invenio_search/templates/search/searchbar_frame_base.html#L29 and removing that line to no longer show the search bar above the search results

Comment (jmartinm) [inline inspirehep/base/templates/search/searchbar_frame_base.html]: I think it should work if the file is named `searchbar_frame.html` and then inside you extend `searchbar_frame_base.html` and just have as content:

```html
{% block body %}
   {% block inner_content %}{% endblock %}
{% endblock %}
```

Can you try?

**Pull request merged**